targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_IMX/gpio_irq_api.c

```diff
@@ -153,8 +155,14 @@ int gpio_irq_init(gpio_irq_t *obj, PinName pin, gpio_irq_handler handler, uint32
             error("gpio_irq only supported on port A-E.");
             break;
     }
-    NVIC_SetVector(gpio_irqs[obj->port], vector);
-    NVIC_EnableIRQ(gpio_irqs[obj->port]);
+
+    int_index = 2 * obj->port;;
```

Comment:
The `;;` can be fixed in another PR since this is good to go.

Reply:
Thanks for finding this, changed